csharp/ParquetSharp.csproj

```
@@ -19,16 +19,13 @@
     <Description>ParquetSharp is a .NET library for reading and writing Parquet files. It's implemented in C# as a PInvoke wrapper around apache-parquet-cpp to provide high performance and compatibility.</Description>
     <Copyright>Copyright G-Research 2021</Copyright>
     <PackageProjectUrl>https://github.com/G-Research/ParquetSharp</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/G-Research/ParquetSharp.git</RepositoryUrl>
     <PackageTags>apache parquet gresearch g-research .net c#</PackageTags>
-    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
+    <PackageLicenseExpression>apache-2.0</PackageLicenseExpression>
```

This is case sensitive, it should be:
```suggestion
    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

---

This got reverted back to apache-2.0 somehow so the build is failing again